tests/test_mpi_vs_multiprocess_vs_single_process.py

```diff
@@ -23,16 +23,17 @@
 from subprocess import check_call, CalledProcessError, run
 import numpy as np
 
+import pyrate.constants
 from pyrate.configuration import Configuration, write_config_file
-from pyrate.core import config as cf
 
 from tests.common import (
     assert_same_files_produced,
     assert_two_dirs_equal,
     manipulate_test_conf,
     MEXICO_CROPA_CONF,
     PY37GDAL304,
-    PY38GDAL304
+    PY38GDAL304,
+    PYTHON3P7
```

Comment:
`PYTHON3P7` has been added to import list, but unused... is this intentional / did some change get missed that was supposed to use this?

Reply:
same effort, probably changed my mind last minute to run the test in a different env.